src/app/components/generate/generate.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+
+import { AngularClass } from '../../models/angular-class.enum';
+import { AngularClassOptions } from '../../models/angular-class-options.interface';
+
+@Component({
+  selector: 'app-generate',
+  templateUrl: './generate.component.html',
+  styleUrls: ['./generate.component.css']
+})
+export class GenerateComponent {
+
+  name = "Name";
+  type: AngularClass;
+  optionalTypes: AngularClassOptions[] = [
+    {id: AngularClass.Component, displayName: 'Component'},
+    {id: AngularClass.Service, displayName: 'Service'}
+  ];
+
+  generateAngularClass(): void {
+    const request = {
+      name: this.name,
+      type: this.type
+    }
+    console.log('request', request);
+  }
+
+  isTypeValid(): boolean {
+    return this.type !>= 0;
```

Comment:
I don't think we'll need this function anymore after the removal of non-selection options. 
but anyway this is much cleaner:
```typescript
isTypeValid(): boolean {
  return this.type < 0;
}
```

`x !>= y` is the same as `x < y`